src/js/exceptions/missing-operand-class-exception.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+angular.module('ngEquation')
+    .factory('MissingOperandClassException', function() {
```

Comment:
Can this be renamed to *Invalid* or *Unknown* rather than "Missing"? I think that would more clearly indicate what the error is.